issuer/src/groups_api.rs

```diff
@@ -1,4 +1,25 @@
 use candid::{CandidType, Deserialize, Principal};
+use std::collections::BTreeMap;
+
+#[derive(Eq, PartialEq, Clone, Debug, CandidType, Deserialize, Ord, PartialOrd)]
+pub enum ArgumentValue {
+    String(String),
+    Int(i32),
+}
+
+#[derive(Clone, Debug, CandidType, Deserialize, Eq, PartialEq, Ord, PartialOrd)]
+pub struct CredentialSpec {
```

Comment:
Will you add a todo to import it from II?

Reply:
Actually, I'm not sure we should use the imported types directly, as these "copies" are a part of public API, which means that any update of `vc_util` types would immediately change the public API of our issuer (hence updating dependencies may become more involved, we just had a similar situation in https://github.com/dfinity/internet-identity/pull/2476).  But I'll add more comments that these are basically copies.

Reply:
Thanks for the explanation!